assets/scripts/app/__tests__/NoConnectionMessage.test.js

```diff
@@ -1,44 +1,51 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
-import { mountWithIntl as mount } from '../../../../test/helpers/intl-enzyme-test-helper.js'
-import { NoConnectionMessage } from '../NoConnectionMessage'
+import { fireEvent } from '@testing-library/react'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
```

Comment:
The first step in porting a test suite from Enzyme to react-testing-library is to change the imports!

In react-testing-library, we no longer need to consider whether to shallow-render or mount. It will always do a full render, including the component's children.

Many, but not all, components in Streetmix assume that there is a Redux or react-intl provider component somewhere in its context. In this case, we previously used a `mountWithIntl` helper which mounts a component wrapped with the react-intl provider. But this component also has Redux in its parent, so although we didn't render with a Redux provider previously, we do that now. So we import a helper function that renders our `<NoConnectionMessage />` component wrapped with both Redux and react-intl providers.